indico/modules/events/registration/blueprint.py

```diff
@@ -230,6 +230,7 @@
 
 # API
 _bp.add_url_rule('/api/registration-forms', 'api_registration_forms', api_misc.RHAPIRegistrationForms)
+_bp.add_url_rule('/api/timetable', 'api_event_timetable', api_misc.RHAPIEventSessionBlocks)
```

Comment:
I would change the URL to `/api/registration/session-blocks` and the endpoint name to `api_session_blocks`. It more clearly describes what it actually returns.